src/dune_rules/ctypes_rules.ml

```diff
@@ -264,8 +214,6 @@ let build_c_program ~foreign_archives_deps ~sctx ~dir ~source_files ~scope
       let+ lib =
         let ctypes = Lib_name.of_string "ctypes" in
         Lib.DB.resolve (Scope.libs scope) (Loc.none, ctypes)
-        (* | Ok lib -> lib | Error _res -> User_error.raise [ Pp.textf "the
-           'ctypes' library needs to be installed to use the ctypes stanza"] *)
```

Comment:
I noticed this leftover when reviewing recent changes internally. Could you delete these two lines?